### DISCUSSION TITLE
Styling and colors

### DISCUSSION THREAD

**Discussion by studiomycel** [DirectoryLister/DirectoryLister#679] [Help & Support] [answered]
Hey,
I've installed DirectoryLister on a local environment and trying to wrap my head around how I would go on about to make changes to the styling.

I understand that you are using Tailwind CSS and one way of editing the colors would be to change the classes.
However, if I were to style the page using this method all changes would be lost when updating.
Is there a proper way to do this that I'm simply missing?

I read in another thread that "In order to change the colors you would need to download the source, make your changes and re-compile the assets".  But I guess that's referring to updating Tailswind's default color codes, or how would I go on about to achieve this?

I also read in the Tailwind Docs that you could create a tailwind.config.js to define any customizations.

Any help in order to understand the workflow would be greatly appriciated 🙂 

All the best,
C

**Comment by PHLAK** [comment, marked as answer]
Directory Lister does not include a way of customizing the styles of the application by the end-user.

To go a bit more in-depth the styles of Directory Lister are pre-compiled. This means there's a step that "builds" the CSS files for use in the final application. Part of the compilation includes removing unused styles. This prevents you from updating the styles in the final application code to anything not-included during the build process.  Also, to your point, changes would be overwritten when updating. Additonally, views are compiled and cached at runtime. Therefor every time you change a class/style in the views you would need to clear the view cache for these changes to be reflected.

**Reply by studiomycel** [reply]
Thank you for your response! That makes sense and answered my question 🙂 

All the best,
Christopher
